src/app/main/main.html

```diff
@@ -66,4 +72,42 @@
     </bookmark-list>
   </modal>
   <dataset-modal></dataset-modal>
+  <tour step="currentStep">
+    <virtual-step
```

Comment:
> I could create a new sub-directory under `components` to store this and the scss

I think that would be my instinct. But again, this may not be a priority :)